0.12/Dockerfile

```diff
@@ -0,0 +1,269 @@
+# This Dockerfile build lnd v0.11.x twice: once on Alpine, once on Debian
+# If the binaries are the same, one is compressed, and copied to the `final` stage
+
+# lnd version to be build
+ARG VERSION=v0.12.0-beta
+
+# Target CPU archtecture of built lnd binary
+ARG ARCH
+
+# Define default versions so that they don't have to be repreated throughout the file
+ARG VER_GO=1.15
+ARG VER_ALPINE=3.12
+
+ARG USER=lnd
+ARG DIR=/data
+
+#
+## NOTE: You should only override the ARGs below, if you know what you're doing
+#
+ARG PKG=github.com/lightningnetwork/lnd
+
+# original content
+#   src: https://github.com/lightningnetwork/lnd/blob/v0.12.0-beta/make/release_flags.mk#L36
+ARG TAGS_BASE="autopilotrpc signrpc walletrpc chainrpc invoicesrpc watchtowerrpc"
+
+# Added by yours truly (@lncm)
+ARG TAGS_EXTRA=""
+
+
+#
+## This stage fetches and verifies source code, and applies necessary fixes
+#
+FROM golang:${VER_GO}-alpine${VER_ALPINE} AS preparer
+
+ARG VERSION
+
+RUN apk add --no-cache  gnupg  git
+
+ENV KEYBASE_USER1=roasbeef
+ENV KEYBASE_USER2=bitconner
+
+# roasbeef and conner keys
+ENV KEYS 4AB7F8DA6FAEBB3B70B1F903BC13F65E2DC84465 9C8D61868A7C492003B2744EE7D737B67FA592C7
```

Comment:
Ah ok, this should be `E4D85299674B2D31FAA1892E372CBD7633C61696`

Reply:
Thanks